life/modernized/05-chapter-5.adoc

```diff
@@ -271,7 +307,8 @@ likely to follow the proposed supplementary enactments.
 To show the injustice of subjecting Friends to the operation of such oppressive laws,
 he declared;
 
-First, That they owned civil government as an ordinance of God,
+[.numbered]
```

Comment:
This whole section here I wasn't sure if I should embed it or not because it seems like it's the narrator quoting and it's not a direct quote, but maybe because of the numbering it should be?